src/api/middleware/company.js

```diff
@@ -12,5 +13,20 @@ const isCompanyRep = (req, res, next) => {
     return next();
 };
 
+const canCreateOffer = async (req, res, next) => {
+    const currentOffers = await (new CompanyService()).getCurrentOffers(req.body.owner);
 
-module.exports = { isCompanyRep };
+    if (currentOffers.length >= CompanyService.MAX_OFFERS_PER_COMPANY) {
+        return res.status(HTTPStatus.BAD_REQUEST).json({
+            reason: `Number of active offers exceeded! The limit is ${CompanyService.MAX_OFFERS_PER_COMPANY} offers`,
```

Comment:
Create a new validation reason in `api/middleware/validators/validationReasons.js` following the schema (lowercase-words-with-hyphen:variable) and use it here and in the tests, so that you can avoid hardcoded strings